iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergInputFormat.java

```diff
@@ -436,49 +526,34 @@ private CloseableIterable<T> openGeneric(FileScanTask task, Schema readSchema) {
 
     @SuppressWarnings("unchecked")
     private CloseableIterable<T> open(FileScanTask currentTask, Schema readSchema) {
-      switch (inMemoryDataModel) {
+      switch (getInMemoryDataModel()) {
```

Comment:
why not just make fields protected to avoid all those getters

Reply:
To avoid checkstyle errors. Checkstyle rules prevent usage of protected variables.